src/shared/seccomp-util.c

```diff
@@ -873,17 +873,42 @@ int seccomp_protect_sysctl(void) {
 }
 
 int seccomp_restrict_address_families(Set *address_families, bool whitelist) {
-
-#if !SECCOMP_RESTRICT_ADDRESS_FAMILIES_BROKEN
```

Comment:
But `test-seccomp` uses `SECCOMP_RESTRICT_ADDRESS_FAMILIES_BROKEN`.
Maybe, we should define a function `is_restrict_address_families_broken` and call it everywhere